.azure/pipelines/ci.yml

```diff
@@ -487,6 +487,26 @@ jobs:
       path: artifacts/TestResults/
       publishOnError: true
 
+# Helix ARM64
```

Comment:
In the past week or so I haven't actually seen the arm tests fail much while tweaking the helix stuff.  I just took a brief manual look at the arm runs for the past few days and I haven't seen any real failures so I'm optimistic.  Maybe we can try turning it on over the weekend and I can monitor things while its lower traffic, its easy to back out anyways if ends up blocking PRs.

Now that the mac queue is gone, and the log typo has been fixed I've actually seen consistent green helix checks since yesterday on my PRs!

Reply:
Let's try this and see how it goes.  We should also send a mail to the team describing the change, and who to talk to if there are issues.